dove/tests/cmd_dove_init_dfinance.rs

```diff
@@ -0,0 +1,224 @@
+#![cfg(test)]
+
+use std::path::{Path, PathBuf};
+use std::fs::{remove_dir_all, read_to_string, create_dir_all};
+use fs_extra::file::write_all;
+use toml::Value;
+use dove::cli::execute;
+
+/// $ dove init -d dfinance
+/// project name: demoproject_43
+#[test]
+fn default() {
```

Comment:
`cmd_dove_init_default` and `cmd_dove_init_dfinance` can be united into a single function to avoid excessive code duplication: they are basically identical